src/extension.ts

```diff
@@ -112,30 +113,45 @@ function tryToForceLanguageId(document: TextDocument, fileName: string, document
         const oldLanguageId: string = document.languageId;
         languages.setTextDocumentLanguage(document, languageId);
         if (notifyUser && !documentCache.includes(document.fileName)) {
```

Comment:
I'm confused about this line. I know it wasn't touched, but with this change `updateLanguageId(..)` always sets its last parameter as `false`. This means that `tryToForceLanguageId` will have the `notifyUser` parameter always `false`. So the question is, when does this code below run, if `notifyUser` is always `false`.

Am I missing some case ?